pkg/runtime/types.go

```diff
@@ -80,17 +86,19 @@ type TypeMeta struct {
 //
 // +protobuf=true
 type RawExtension struct {
-	// RawJSON is the underlying serialization of this object.
-	RawJSON []byte
+	// Raw is the underlying serialization of this object.
```

Comment:
So maybe I don't fully understand it.
My rationale for having it is that it seemed to be that in some cases there wasn't any way to infer that information if we don't have them in RawExtension - as examples see above:
- pkg/api/meta/help.go 
- pkg/runtime/embedded.go (conversions between RawExtension and Object)

But maybe I'm missing how RawExtensions generally works...